src/templates/default_nativeapp/app/setup_script.sql

```diff
@@ -0,0 +1,48 @@
+CREATE OR REPLACE APPLICATION ROLE app_instance_role;
```

Comment:
The case of sql file is decided by examples taken from https://docs.snowflake.com/en/developer-guide/native-apps/creating-setup-script. We should stay consistent with the official snowflake documentation.